### ISSUE TITLE
A11y_SQL database projects extension_Create Project From Database_Create Project_Status Message: NVDA is not announcing the loading message which is appearing after activating the create button.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/azuredatastudio#22158]
### GitHub Tags
#A11yTCS; #A11ySev2; #DesktopApp; #Status Message; #WCAG4.1.3; #Win11; #SH-SQL Azure Data Studio-Win32-March23; #Win32; #SQL Azure Data Studio; #NVDA; #A11yWCAG2.1; #Screenreader; #Closed; #Regressed:03-30-23;

### Environment Details
Application Name: SQL Azure Data Studio
Version: 1.41.2 (user setup)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Screen reader: NVDA

### Repro Steps

1. Turn on NVDA.
2. Launch Azure Data Studio
3. Tab till extension and search for “SQL Database Projects” and install it.
4. Tab till database project and press enter. “Database Project” panel will open.
5. Tab till “View and More Action” button and press enter.
6. Tab till “Create Project From Database” and press enter.
7. Tab till server field and press enter. “Connection” dialog will open. Fill the details for SQL server and connect.
8. Tab till Create button and press enter.
9. Verify whether the NVDA is announcing the loading message which is appearing after activating the create button or not.

### Actual Result
NVDA is not announcing the loading message which is appearing after activating the create button.

### Note
Similar issue observed in the below screen.
1. Database project->Publish project-> Generate script

### Expected Result
NVDA should announce the loading message which is appearing after activating the create button.

### User Impact
Screen reader user will not be able to get the proper information like whether the data base is created or still it is creating or not.

### Attachment
![Status Message](https://user-images.githubusercontent.com/99243079/223075235-2e50a994-3c85-4cf0-9430-66062fd1dced.gif)
![Presentation6](https://user-images.githubusercontent.com/99243079/223625944-7ca3f351-3d38-4b78-9a8a-ea8d059b2603.gif)

**Comment by kisantia** [comment]
@Samhitha66 this has been fixed in the latest version of the sql database projects extension (0.23.0). Please download the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and verify this is fixed. Thanks!
